src/anjali/A13.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Assignment - 13 : 28th Dec'2021
+
+Write a program to satisfy below requirement using indexOf and lastIndexOf method.
+
+1. Print all duplicate characters from given string.
+2. Print all unique characters from given string.
+3. Return first unique character from given string.
+4. Return first duplicate character from given string.     
+*/
+package anjali;
+
+import java.util.Scanner;
+
+public class A13 {
+	
+	void printAlluniqueCharString(String name){
+		for(int index=0;index<name.length();index++){
+			char ch= name.charAt(index);
+			if(name.indexOf(ch)==name.lastIndexOf(ch)) 
+				System.out.println(ch);
+		}
+	}	
+	
+	void printAllDuplicateCharString(String name){
+		String z="";  
+		char ch=' ';
+		for(int index=0;index<name.length();index++){
+			if(!z.contains(String.valueOf(name.charAt(index))))
+			ch= name.charAt(index);
```

Comment:
line 29 & 30 is combined if you write code in this way you should write code inside this { }.